whylog/config/parsers.py

```diff
@@ -67,9 +67,12 @@ def from_dao(cls, serialized_parser):
 class RegexParserFactory(object):
     @classmethod
     def create_from_intent(cls, parser_intent):
+        convertions = dict(
+            (group_id, group.converter) for (group_id, group) in parser_intent.groups.items()
+        )
```

Comment:
In this way we lost sample group content, which you add in this pull request. To RegexParser constructor should be passed parser_intent.groups.